crawler/helpers/general_helpers.py

```diff
@@ -254,7 +261,7 @@ def parse_decimal128(value: Decimal128) -> Optional[Decimal]:
         Decimal -- converted number
     """
     try:
-        return value.to_decimal()
+        return Decimal(value.to_decimal())
```

Comment:
The cast function is just for typing, it doesn't do anything with the argument, simply passes it through but type checkers know what type the argument then is.

Reply:
is value.to_decimal() not a Decimal already? If this is an error by the syntax checker, why is casted with Decimal instead of annotating the value.to_decimal as a Decimal for the syntax checker with cast()

Reply:
Sorry! I did not read your first question properly, I have used `cast()` instead, that makes more sense.